src/dev/impl/DevToys/Strings/en-US/XmlFormatter.resw

```diff
@@ -0,0 +1,159 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please duplicate this file in every other language folder (but no need to translate them). It will help future translators to know what is and what isn't translated yet.

Reply:
Done.